scripts/core/notify/notify.tsx

```diff
@@ -132,3 +133,20 @@ export default angular.module('superdesk.core.notify', ['superdesk.core.translat
         // eslint-disable-next-line
         return ReactDOM.render(<NotifyComponent />, targetEl); // returns instance of NotifyComponent
     }]);
+
+export const notify = {
+    info: (text: string, displayDuration?: number, options?: any) => {
+        ng.get('notify').info(text, displayDuration, options);
+    },
+    success: (text: string, displayDuration?: number, options?: any) => {
+        // eslint-disable-next-line angular/no-http-callback
```

Comment:
why does this rule need to be disabled on `success` and `error` but not on `warning` and `info`?

Reply:
I think this is a bug with the linting, for some reason it saw this object as an angular http callback....